models/shopping-list.js

```diff
@@ -9,6 +9,11 @@ const shoppingListSchema = new mongoose.Schema(
     name: { type: String, required: true },
     store: { type: mongoose.Schema.Types.ObjectId, ref: 'Store' },
     items: { type: [ShoppingListItem], default: [] },
+    userId: {
```

Comment:
Please rename to `user` rather than `userId`.

The reason is that we will probably want to call `.populate()` and then Mongoose will turn this property into a full user object. So (right now), subsequent accesses of the user would look weird: `list.userId.id` for example. If we set this to be `user` that would be much clearer: `list.user.id`.

Reply:
We can change this on our branch, however, since it doesn't look like your code relies on it.